wd_core.au3

```diff
@@ -246,6 +247,10 @@ Func _WD_CreateSession($sCapabilities = Default)
 		EndIf
 	Else
 		$iErr = $_WD_ERROR_Exception
+
+		If Json_Get($oJSON, "[value][error]") = $_WD_ErrorSessionNotCreated Then
```

Comment:
I would like to do this:
```autoIt
If Json_Get($oJSON, "[value][error]") Then
```
beacuse `__WD_Error` in case of `$_WD_DEBUG_Info`  will show us any error information.

Reply:
It looks good now.
The only new changes I do are in: [header + comments](https://github.com/Danp2/au3WebDriver/pull/428/commits/d3639e5b9e67d90e366add653175c16ff2348647)